subsys/net/ip/tcp.c

```diff
@@ -51,6 +51,7 @@ static struct k_work_q tcp_work_q;
 static K_KERNEL_STACK_DEFINE(work_q_stack, CONFIG_NET_TCP_WORKQ_STACK_SIZE);
```

Comment:
Typo in commit `net: tcp: Retrigger the send work in case of loopback`
s/exectue/execute/